client/dashboard/profile-wizard/steps/start/index.js

```diff
@@ -41,42 +41,58 @@ class Start extends Component {
 	}
 
 	componentDidMount() {
-		const { updateProfileItems } = this.props;
+		const { updateProfileItems, profileItems } = this.props;
 		if (
 			this.props.activePlugins.includes( 'jetpack' ) &&
 			this.props.activePlugins.includes( 'woocommerce-services' )
 		) {
-			updateProfileItems( { wcs_jetpack: 'already-installed' } );
+			// Don't track event again if they revisit the start page.
+			if ( 'already-installed' !== profileItems.plugins ) {
+				recordEvent( 'wcadmin_storeprofiler_already_installed_plugins', {} );
+			}
+
+			updateProfileItems( { plugins: 'already-installed' } );
 			return updateQueryString( { step: 'store-details' } );
 		}
 	}
 
 	async skipWizard() {
-		const { createNotice, isProfileItemsError, updateProfileItems } = this.props;
+		const { createNotice, isProfileItemsError, updateProfileItems, activePlugins } = this.props;
 
-		await updateProfileItems( { skipped: true, wcs_jetpack: 'skipped' } );
+		const plugins = activePlugins.includes( 'jetpack' ) ? 'skipped-wcs' : 'skipped';
```

Comment:
I may be overthinking this, but this won't track if WCS is installed and Jetpack is skipped, right?

Maybe we just keep the `skipped` boolean setting (or update to `skipped_plugins`) and just record the items that are being skipped in tracks.  What do you think?

Reply:
> Could you check against this.props.activePlugins or would that not work?

This won't work. I need to know that the page was skipped automatically, so that we show the usage tracking modal on the store details step instead: https://github.com/woocommerce/woocommerce-admin/pull/3040/files#diff-5965f2cd86a39747c2fb0ce9d391e444. I can't rely on activePlugins there because it should always contain both by the time you get to the step.

> I may be overthinking this, but this won't track if WCS is installed and Jetpack is skipped, right?

Correct, but we are not handling WCS being installed without Jetpack at all right now. If Jetpack is not installed, we end up showing both sets of benefits -- so `skipped` would apply here. I thought there was a Slack conversation about this, but I can't seem to find it now. But I think the gist is that we should expect this to be an edge case. WooCommerce Services doesn't do anything without Jetpack except show some banners.

> Maybe we just keep the skipped boolean setting (or update to skipped_plugins) and just record the items that are being skipped in tracks. What do you think?

I still need to keep some form of `plugins` == 'already-installed' logic. I think the current enum logic does a good job of telling which page the user ends up seeing based on the different flows we currently have implemented, and allows us to show the modal later if necessary. I could separate this out to two different pieces of meta though, if you think a bool is better here.

Reply:
>WooCommerce Services doesn't do anything without Jetpack except show some banners. To simplify our logic, we show both plugins if Jetpack isn't present. Reactivating WCS on the activate step doesn't harm anything either.

That makes a lot of sense.  Thanks for the explanation!

>I could separate this out to two different pieces of meta though, if you think a bool is better here.

I think you're right.  Let's keep it as is and we can iterate if necessary.